src/tpc/google-analytics.ts

```diff
@@ -0,0 +1,54 @@
+import { addImports, addTemplate } from '@nuxt/kit'
+import { GoogleAnalytics, GoogleTagManager } from 'third-party-capital'
+import type { RegistryScript } from '../runtime/types'
+import { getTpcScriptContent } from './utils'
+
+export default async function addGoogleAnalytics() {
+  const { dst, filename } = addTemplate({
+    write: true,
+    getContents() {
+      return getTpcScriptContent({
+        data: GoogleAnalytics({}),
```

Comment:
since we're not passing in any values here... would it be useful for tpc to export the raw JSON files?
we could also expose the formatData function if that's helpful in some way.

Reply:
Yeah I was thinking about asking that